src/_gettsim_tests/test_data/arbeitsl_geld_2/2023-01/hh_id_25.yaml

```diff
@@ -0,0 +1,158 @@
+---
+info:
+  note: >-
+    Test with old income thresholds with income from self employment, adjusted for
+    Kindersofortzuschlag. Constructed test based on hh 19.
+  source: ''
```

Comment:
You didn't find an online calculator which allowed for self-employed income? If not, it should be fine as it is.

Reply:
I couldn't find one, I think since the rules are the same, those usually distinguish between different sources of income.